custom-config-server-client/pom.xml

```diff
@@ -1,69 +1,62 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.3.3.RELEASE</version>
-		<relativePath/> <!-- lookup parent from repository -->
+		<version>3.2.0</version>
 	</parent>
 	<groupId>com.microsoft.sample</groupId>
 	<artifactId>custom-config-server-client</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
 	<name>custom-config-server-client</name>
-	<description>Demo project for showing how to access Azure Spring Apps managed Config Server through Azure RBAC when your applications are running outside Azure Spring Apps</description>
-
+	<description>Demo project for showing how to access Azure Spring Apps
+		managed Config Server through Azure RBAC when your applications are
+		running outside Azure Spring Apps</description>
 	<properties>
-		<java.version>1.8</java.version>
+		<java.version>17</java.version>
+		<spring-cloud.version>2023.0.0</spring-cloud.version>
 	</properties>
-
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
-
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-config</artifactId>
-			<version>2.2.5.RELEASE</version>
 		</dependency>
-
 		<dependency>
-			<groupId>com.microsoft.azure</groupId>
-			<artifactId>azure</artifactId>
-			<version>1.34.0</version>
+			<groupId>com.azure</groupId>
+			<artifactId>azure-core-management</artifactId>
+			<version>1.11.9</version>
 		</dependency>
-
-		<dependency>
-			<groupId>com.microsoft.azure</groupId>
-			<artifactId>azure-mgmt-graph-rbac</artifactId>
-			<version>1.34.0</version>
-		</dependency>
-
 		<dependency>
 			<groupId>com.azure</groupId>
-			<artifactId>azure-core</artifactId>
-			<version>1.3.0</version>
+			<artifactId>azure-identity</artifactId>
+			<version>1.11.1</version>
+			<scope>compile</scope>
 		</dependency>
-
 		<!-- tag::actuator[] -->
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
 		<!-- end::actuator[] -->
-
-		<!-- tag::tests[] -->
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
-		</dependency>
-		<!-- end::tests[] -->
 	</dependencies>
-
+	<dependencyManagement>
```

Comment:
Better use the azure-sdk-bom or Spring Cloud Azure BOM, so we don't need to specify the version for azure-identity or azure-core-management